docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "http instrument pyroscope"
+descriptiontion: "k6 pyroscope instrumentation API"
+weight: 04
+weight: 04
+---
+
+# Pyroscope
+
+
+With this jslib, you can _instrument_ HTTP requests so that Grafana Cloud Profiles can match profiles to Grafana Cloud K6 tests.
+
+## About baggage header
+
+A _barrage header_ is a standardized HTTP header to propagate distributed context. The [W3C specification](https://www.w3.org/TR/baggage/) goes into more details on the specifics, but like many other headers is a list of key, value  pairs. 
```

Comment:
Consider removing the emphasis on barrage header. Also, on the second sentence, barrage header is missing the closing _ for the emphasis (which should probably be removed since we don't tend to use emphasis that often).